developer/contributing/pull-requests.rst

```diff
@@ -32,67 +54,105 @@ If you are a member of the `Sulu organization`_ you should also:
 Template
 --------
 
-Template is divided in two parts: *tasks* (for developer) and *information*
-(for reviewer).
+When creating a pull request on GitHub, there is already a pre filled template
+for you. The template is divided in several parts.
+
+Checklist
+`````````
+The pull request must include the following checklist at the top to ensure that
+contributions may be reviewed without needless feedback loops and that your
+contributions can be included into Sulu as quickly as possible:
+
+An example submission could now look as follows:
 
-The information should include the following fields:
+.. code-block:: text
 
-* **Fixed tickets**: **list of tickets**; Tickets to close after merge. These
-  should be of the form ``fixes #123, fixes #321``.
-* **Related PRs**: **link to PRs`**, PRs that are required for the current one
-* **BC breaks**: **none or description**; any compatibility breaking changes
-* **Documentation PR**: **link to PR**; Link to the documentation PR
-  (if applicable)
+    | Q                | A
+    | ---------------- | ---
+    | Bug fix?         | yes
+    | New feature?     | no
+    | BC breaks?       | no
+    | Deprecations?    | yes
+    | Fixed tickets    | fixes #1243, fixes #1443
+    | Related PRs      | none
+    | License          | MIT
+    | Documentation PR | sulu-io/sulu-docs#153
+
+Some answers to the questions trigger some more requirements:
+
+* If you answer yes to "Bug fix?", check if the bug is already listed in the
+  Sulu issues and reference it/them in "Fixed tickets". These should be of the
+  form ``fixes #123, fixes #321``.
+* If you answer yes to "New feature?", you must submit a pull request to the
+  documentation and reference it under the "Documentation PR" section.
+* If you answer yes to "BC breaks?", the patch must contain updates to the
+  ``UPGRADE.md`` file.
+* If you answer yes to "Deprecations?", the patch must contain updates to the
+  ``UPGRADE.md`` file.
```

Comment:
I would replace `patch` with `pull request` in the previous two bullet points. Using a different term here for the same thing might be confusing.